src/media/css/elements/nav/index.styl

```diff
@@ -0,0 +1,212 @@
+@require('../../constants/typography');
+@require('constants');
+
+mkt-nav {
+  border-right: 2px solid rgba(0, 0, 0, 0.2);
+  fixed: top 0 left 0;
+  font-size: 18px;
+  height: 100%;
+  overflow-y: scroll;
+  transition: transform cubic-bezier(0.86, 0, 0.07, 1) 333ms;
+  width: 260px;
+  z-index: $zindex--nav-hamburger-root;
+
+  li {
+    margin: 25px 0;
+  }
+
+  a {
```

Comment:
Elaborate on your concerns? I consciously did it this way to keep specificity down so implementations could easily overwrite as appropriate.